src/unclutter.c

```diff
@@ -93,6 +94,7 @@ static void parse_args(int argc, char *argv[]) {
         { "jitter", required_argument, 0, 0 },
         { "exclude-root", no_argument, 0, 0 },
         { "ignore-scrolling", no_argument, 0, 0 },
+        { "touch", no_argument, 0, 0 },
```

Comment:
"touch" somewhat suggests that it simply includes touch events, but this isn't the case; rather, it does the opposite of what we do for other input events. So I think it'd be better if we could name this `--hide-on-touch` (and everywhere else accordingly)